pkg/apis/keda/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,27 +1,19 @@
 // +build !ignore_autogenerated
 
 /*
-MIT License
-
-Copyright (c) Microsoft Corporation. All rights reserved.
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
I don't think this is on purpose? :thinking:

Reply:
@tomkerkhove this code is auto generated by the kubernetes code generator.

Reply:
When generating, make sure to include [`--go-header-file`](https://github.com/kedacore/keda/blob/master/hack/generate-groups.sh#L12)

See this PR for context https://github.com/kedacore/keda/pull/144

Reply:
@ahmelsayed I couldn't find a target in the Makefile that can generate the code. Would you be able to send me the command? When I generate the code I get tons of errors in the test - too many arguments in call to testing.NewPatchSubresourceAction

Reply:
You're right, the makefile is lacking and the dependencies are not broken for code gen. I opened https://github.com/kedacore/keda/pull/349 with the fix. 

If you're unable to merge that, change `code-generator` in Gopkg.toml to `1.12.3`, then run:

```bash
dep ensure -v
./hack/generate-groups.sh
git checkout -- vendor/
```

I think